src/shipping/magento-shipping-quick-look.md

```diff
@@ -2,38 +2,38 @@
 title: Quick Look
 ---
 
-## [Advanced Shipments]({{ site.baseurl }}{% link sales/shipments-create.md %})
+## Advanced Shipments
 
-Magento Shipping is now available in [seventeen countries]({{ site.baseurl }}{% link shipping/magento-shipping.md %}), where you can create shipments with advanced features such as route and package information with dimensional weight, carrier quotes, international details with required export information, harmonization codes, shipping documents and labels, delivery instructions, and tracking information.
+Magento Shipping is now available in [seventeen countries]({{ site.baseurl }}{% link shipping/magento-shipping.md %}), where you can create [shipments]({{ site.baseurl }}{% link sales/shipments-create.md %}) with advanced features such as route and package information with dimensional weight, carrier quotes, international details with required export information, harmonization codes, shipping documents and labels, delivery instructions, and tracking information.
 
-## [Locations]({{ site.baseurl }}{% link shipping/magento-shipping-locations.md %})
+## Locations
 
-Configure multiple warehouses, distribution centers, retail stores, drop shippers, and third-party logistics. Allocate shipments to any supported fulfillment location.
+Configure multiple warehouses, distribution centers, retail stores, drop shippers, and third-party logistics. Allocate shipments to any supported fulfillment [location]({{ site.baseurl }}{% link shipping/magento-shipping-locations.md %}).
 
-## [Click &amp; Collect]({{ site.baseurl }}{% link shipping/magento-shipping-locations.md %})
+## Click &amp; Collect
 
-Offer the convenience of ordering online, with convenient customer pickup at a nearby location.[!
+Offer the convenience of ordering online, with convenient customer pickup at a [nearby location]({{ site.baseurl }}{% link shipping/magento-shipping-locations.md %}).
 
-## [Collection Points]({{ site.baseurl }}{% link shipping/magento-shipping-collection-points.md %})
+## Collection Points
 
-Collection points allow customers to order online and have it shipped to a designated collection point, rather than to their address.
+[Collection points]({{ site.baseurl }}{% link shipping/magento-shipping-collection-points.md %}) allow customers to order online and have it shipped to a designated collection point, rather than to their address.
 
-## [Dispatches]({{ site.baseurl }}{% link sales/dispatches.md %})
+## Dispatches
```

Comment:
Add the link in this paragraph that you took from the heading

Reply:
**done**